packages/styles/scss/components/content-item-horizontal-thumbnail/_content-item-horizontal-thumbnail.scss

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
```

Comment:
don't think we need to have a separate scss file in the styles package, can you move this to the scss file you created within the content-item-horizontal web-components folder? It's a little confusing to have 2 different files.

Reply:
@annawen1 I was using the `<dds-content-item-horizontal-media>` component/variation as an example, which had a setup similar to what I created. But I can move these styles over as you suggest!

Reply:
Also, I totally did make them duplicate. 🙈